src/Orchard.Web/Modules/Orchard.Azure/Services/TaskLease/AzureApplicationEnvironment.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Microsoft.WindowsAzure.ServiceRuntime;
+using Orchard.Environment;
+
+namespace Orchard.Azure.Services.TaskLease {
```

Comment:
The namespace is wrong, it should not mention TaskLease anymore but maybe Environment.